src/i18n/en.js

```diff
@@ -6,6 +6,7 @@ export const en = {
   matchMeToBenefits: "Match me to Benefits",
   allEligibleBenefitsHeader: "All Benefits",
   eligibleBenefitsHeader: "Benefits that apply to you",
```

Comment:
I would just change this to Service Canada benefits that apply to you

Reply:
Done